### PR TITLE
added hippie hollow

### PR DESCRIPTION
**Pull request by tamistcRackSpace**

### CODE REVIEW

File: tamistcRackspace-austin.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 - Snap Kitchen (Local Organic)
 
 ### Secret spots for locals to do
-
+-hippie hollow
```
Comment (githubteacher) [inline tamistcRackspace-austin.md]: What in the world is `hippie hollow` @tamistcRackSpace? Sounds fun!

Comment (tamistcRackSpace) [inline tamistcRackspace-austin.md]: Totally fun :)

**Comment by tamistcRackSpace** [comment]
trying to merge on information super highway of github flow

**Comment by tamistcRackSpace** [comment]
Thought I added some more stuff

**Pull request merged**